doc/cpp_interface.md

```diff
@@ -3,15 +3,39 @@
 
 ####  Compile C++ sources and run tests 
 
-To compile the C++ code you need the [lp_solve](http://lpsolve.sourceforge.net/5.5/) library. For example, for Unix/Linux you need `liblpsolve55.so` (this is available from the library's [webpage](http://lpsolve.sourceforge.net/5.5/) as well as a package in several linux distributions e.g. [debian](https://packages.debian.org/stretch/liblpsolve55-dev)). You have to specify the path to `liblpsolve55.so`, by running, in folder test:  
-```
-cmake -DLP_SOLVE=_PATH_TO_LIB_FILE_ .  
-make  
+To compile the C++ code you need the [lp_solve](http://lpsolve.sourceforge.net/5.5/) library. For example, for Unix/Linux you need `liblpsolve55.so`. This is available from the library's [webpage](http://lpsolve.sourceforge.net/5.5/) as well as a package in several linux distributions e.g. [debian](https://packages.debian.org/stretch/liblpsolve55-dev) `sudo apt-get install lp-solve`. 
+
+You have to specify the path to `liblpsolve55.so/dll/dylib`, by running, in folder test:  
+```bash
+mkdir -p test/build && cd test/build
+cmake -DLP_SOLVE=_PATH_TO_LIB_FILE_ ..
+make
 ```
 For example:  `-DLP_SOLVE=/usr/lib/lpsolve/liblpsolve55.so`  
 
 You can run the tests by `cmake test` or `ctest -jK` where `K` the number of `CPU` threads. By adding the option `--verbose` to `ctest` you get more information about the tests, *e.g.* time per test, volume computed and the name of the polytope or convex body. 
 
+##### Development environment from Docker container
```

Comment:
Thanks! You could also add a link on how to get started with docker or something?

Reply:
Done!